src/nvim/msgpack_rpc/channel.c

```diff
@@ -41,6 +41,8 @@
 static PMap(cstr_t) *event_strings = NULL;
 static msgpack_sbuffer out_buffer;
 
+extern MultiQueue *ch_before_blocking_events;
```

Comment:
This is a regression, now the sharing is implicit rather than explicit. Better to keep it in the `.h` file and use the EXTERN/INIT pattern (and make sure it is included in main.c to make the INIT work).

Reply:
Lets see if this works. Thanks.